### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by sarah-adam**
Added a example bucket path: cloudflare-logs/http_requests/example.com

### CODE REVIEW

File: content/logs/get-started/enable-destinations/r2/index.md [hunk 1]
```diff
@@ -56,7 +56,9 @@ Enable Logpush to R2 via the dashboard.
 7.  Select R2.
 
 8.  Enter the following destination information:
-    - Bucket path
+    - Bucket path - For example,
```
Comment (victor-perov) [inline content/logs/get-started/enable-destinations/r2/index.md]:
```suggestion
    - Bucket path, for example, `cloudflare-logs/http_requests/example.com`
```

**Review by victor-perov — CHANGES_REQUESTED** [review]
I think we can move an example to the same line and wrap it in a single "`".

**Review by cmackenzie1 — APPROVED** [review]
Other than the comment from Victor, it looks good to me!

File: content/logs/get-started/enable-destinations/r2/index.md [hunk 2]
```diff
@@ -56,7 +56,9 @@ Enable Logpush to R2 via the dashboard.
 7.  Select R2.
 
 8.  Enter the following destination information:
-    - Bucket path
+    - Bucket path, for example, `cloudflare-logs/http_requests/example.com`
+```cloudflare-logs/http_requests/example.com```
```
Comment (victor-perov) [inline content/logs/get-started/enable-destinations/r2/index.md]:
```suggestion
```

File: content/logs/get-started/enable-destinations/r2/index.md [hunk 3]
```diff
@@ -56,7 +56,9 @@ Enable Logpush to R2 via the dashboard.
...
+
```
Comment (victor-perov) [inline content/logs/get-started/enable-destinations/r2/index.md]:
```suggestion
```

**Review by victor-perov — CHANGES_REQUESTED** [review]
Just a tiny cleanup needed.

**Review by angelampcosta — APPROVED** [review]

**Review by victor-perov — APPROVED** [review]

**Comment by angelampcosta** [comment]
@sarah-adam thank you so much for your feedback!

**Pull request merged**